pint_xarray/testing.py

```diff
@@ -0,0 +1,22 @@
+from . import conversion
+
+
+def assert_units_equal(a, b):
+    """ assert that the units of two objects are equal
+
+    Raises an :py:exc:`AssertionError` if the units of both objects are not
+    equal. ``units`` attributes and attached unit objects are treated
+    differently.
```

Comment:
Might be good to explain what is meant by "treated differently"?

Reply:
How about
```suggestion
    equal. ``units`` attributes and attached unit objects are compared
    separately.
```

Reply:
Looks good! I'll commit the suggestion and merge.